request-cert/k8s_certificates.go

```diff
@@ -22,6 +22,9 @@ import (
 	"time"
 
 	"github.com/pkg/errors"
+
```

Comment:
Kill whitespace here and on line 27.

Reply:
It's actually a lspace b/w different groups of imports. Std imports, github/* and then k8s.io/* in this case. I can remove if necessary.